docs/release-notes-v11.16.md

```diff
@@ -0,0 +1,60 @@
+installs PostgreSQL and complements it by a selection of extensions that
+enable solving essential practical tasks efficiently.
+
+
+This release is based on [PostgreSQL
+11.16](https://www.postgresql.org/docs/release/11.16/). The set of extensions supplied with Percona Distribution for PostgreSQL now includes the [HAProxy](http://www.haproxy.org/) - a high-availability and load-balancing solution.
+
+| Extension           | Version        | Description                  |
+| ------------------- | -------------- | ---------------------------- |
+| [Patroni](https://patroni.readthedocs.io/en/latest/) | 2.1.3 | a HA (High Availability) solution for PostgreSQL |
+| [PgAudit](https://www.pgaudit.org/)             | 1.3.4   | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
+| [pgAudit set_user](https://github.com/pgaudit/set_user)| 3.0.0 | provides an additional layer of logging and control when unprivileged users must escalate themselves to superusers or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.38    | a backup and restore solution for PostgreSQL       |
+|[pgBadger](https://github.com/darold/pgbadger)   | 11.8     | a fast PostgreSQL Log Analyzer.|
+|[PgBouncer](https://www.pgbouncer.org/)          |1.17.0    | a lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.7   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)                                         |1.0.0 | collects and aggregates statistics for PostgreSQL and provides histogram information.|
+|[wal2json](https://github.com/eulerto/wal2json)  |2.4       | a PostgreSQL logical decoding JSON output plugin|
+|[HAProxy](http://www.haproxy.org/) | 2.5.6 | a high-availability and load-balancing solution |
+
+Percona Distribution for PostgreSQL also includes the following packages:
+
+- `llvm` 12.0.1 packages for Red Hat Enterprise Linux 8  and derivatives. This fixes compatibility issues with LLVM from upstream.
+- supplemental ETCD packages which can be used for setting up Patroni clusters. These packages are available for the following operating systems:
+
+|  Operating System   |Package               | Version | Description   |
+| ------------------- | ---------------------| --------|---------------|
+| RHEL 7             |`python3-python-etcd` | 0.4.5   | A Python client for ETCD     |
+| RHEL 8             | `etcd`               | 3.3.27  | A consistent, distributed key-value store|
+|                     | `python3-python-etcd`| 0.4.5   | A Python client for ETCD     |
+| Debian 9 ('stretch')| `etcd`               | 3.3.27  | A consistent, distributed key-value store|
+|                     | `python3-etcd`       | 0.4.5   | A Python client for ETCD     |
+
```

Comment:
etc version will be 3.3.11 and python-etcd 0.4.3 - so it stays as is. We're working on updating it for the next release